AsyncDisplayKit/ASCollectionView.h

```diff
@@ -406,6 +406,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)shouldBatchFetchForCollectionView:(ASCollectionView *)collectionView;
 
+/**
+ * Use `collectionView:didEndDisplayingNode:forItemAtIndexPath:` instead.
```

Comment:
Is this comment finished?

Can you add didEndDisplayingCell: as an asserted not-to-implement method in the delegate proxy if it isn't already?  I'm pretty sure it is already there, just want to be sure.